.circleci/config.yml

```diff
@@ -72,14 +78,13 @@ jobs:
           name: Run Go Azure tests
           command: |
             export PREFIX="$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 5 | head -n 1 | sed -E 's/^([0-9]+)(.*)/\2\1/g')"
-            VERSION=$(git describe --tags --abbrev=8 --exact-match 2>/dev/null || echo "${POLCADOT_PROVIDER_VERSION}")
-            [[ "${VERSION}" =~ "^azure" ]] || VERSION="${POLCADOT_PROVIDER_VERSION}"
-            export VERSION=${VERSION#*-}
-            make test-azure
-          environment:
-            - POLCADOT_PROVIDER_VERSION: 0.1.0
+            VERSION=$(git describe --tags --abbrev=8 --exact-match 2>/dev/null || echo "${AZURE_POLCADOT_PROVIDER_VERSION}")
```

Comment:
It's polKadot. Please, make sure you have fixed it everywhere in the code

Reply:
Fixed